tests/mcs_benchmark_data_test/cli/create_benchmark_pipeline_test.py

```diff
@@ -17,53 +19,16 @@ def test_create_benchmark_pipeline(tmpdir):
 
     test_args = TestArgs("snazzy_new_benchmark", True)
 
-    CreateBenchmarkPipelineCommand().make_benchmark_directories(
+    CreateBenchmarkPipelineCommand()._make_benchmark_directories(

```

Comment:
The test should run the code as the CLI will run it, and not call private/protected methods. So this should be 
```
CreateBenchmarkPipelineCommand(args=test_args, root_path=tmpdir)()
```